docs/zh/latest/plugins/jwt-auth.md

```diff
@@ -48,6 +53,9 @@ title: jwt-auth
 | algorithm     | string  | 可选   | "HS256" | ["HS256", "HS512", "RS256"] | 加密算法                                                                                                      |
 | exp           | integer | 可选   | 86400   | [1,...]                     | token 的超时时间                                                                                              |
 | base64_secret | boolean | 可选   | false   |                             | 密钥是否为 base64 编码                                                                                        |
+| vault | object | 可选    |    |                             | Vault 是否用于密钥（HS256/HS512 的密钥或 RS256 的公钥和私钥）的存储和检索。该插件默认使用的 Vault 路径为 `kv/apisix/consumer/<consumer name>/jwt-auth` 用于检索密钥 |
```

Comment:
Strange. I can't actually get the point of it.

Reply:
```suggestion
| vault | object | 可选    |    |                             | Vault 是否用于密钥（HS256/HS512 的密钥或 RS256 的公钥和私钥）的存储和检索。该插件默认使用 vault 路径为 `kv/apisix/consumer/<consumer name>/jwt-auth` 进行密钥检索。 |
```